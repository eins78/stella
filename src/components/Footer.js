import React from 'react'

const Footer = () => (
  <footer>
    Â© Max F. Albrecht 2014 | Feed: <a className='feed-rss' href='../../?/feed/'>RSS</a>, <a className='feed-json' href='../../?/json/'>JSON</a> | View: <a href='./'>default</a>, <a href='./?dark'>dark</a>, <a href='./?plain'>plain</a><br />
  </footer>
)

export default Footer
