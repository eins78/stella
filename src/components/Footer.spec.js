/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import * as fixtures from '../../spec/fixtures.js'

import Footer from './Footer'

it('renders', () => {
  const component = renderer.create(
    <Footer site={fixtures.site} />
  )
  expect(component.toJSON()).toMatchSnapshot()
})
