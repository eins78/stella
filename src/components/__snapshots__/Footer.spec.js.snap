exports[`test renders 1`] = `
<footer>
  Â© Max F. Albrecht 2014 | Feed: 
  <a
    className="feed-rss"
    href="../../?/feed/">
    RSS
  </a>
  , 
  <a
    className="feed-json"
    href="../../?/json/">
    JSON
  </a>
   | View: 
  <a
    href="./">
    default
  </a>
  , 
  <a
    href="./?dark">
    dark
  </a>
  , 
  <a
    href="./?plain">
    plain
  </a>
  <br />
</footer>
`;
